### PR TITLE
Updates to profile syntax

### PR DESCRIPTION
**Pull request by jhaag75**
Made corrections to IRIs, syntax, and concepts. Did not fix statement templates and patterns. Need to meet to further discuss your goals before I can help with the templates and patterns.

### CODE REVIEW

**Review by kirstykitto — COMMENTED** [review]
On reflection today I think this will work. We should be able to use the verbs etc. that are deleted here in the specification of the patterns right? If so then I will approve and go to work on them once I have merged this pull request.

**Review by kirstykitto — APPROVED** [review]
Accepted.

**Comment by jhaag75** [comment]
Exactly! Yes, since the verbs are referenced in the templates that's the
only place they need to be if you are reusing an existing IRI. If you are
creating a new IRI, then it is declared as a concept associated with your
profile. This was intentional in the profile spec, so only one authority /
CoP would be responsible for maintaining verbs, etc. If concepts are
declared in multiple profiles then you run into a possible collision where
multiple people could update or create duplicates with conflicting meaning
or metadata.

**Pull request merged**
